src/fastfield/multivalued/reader.rs

```diff
@@ -46,6 +46,24 @@ impl<Item: FastValue> MultiValuedFastFieldReader<Item> {
         self.vals_reader.get_range_u64(range.start, &mut vals[..]);
     }
 
+    /// Returns the minimum value for this fast field.
+    ///
+    /// The max value does not take in account of possible
+    /// deleted document, and should be considered as an upper bound
+    /// of the actual maximum value.
+    pub fn min_value(&self) -> Item {
+        self.vals_reader.max_value()
```

Comment:
out-of-context, is it deliberate to have `min_value()` return `vals_reader.max_value()` and vice versa?

Reply:
Thanks for catching that. @evanxg852000 can you fix?

Reply:
good catch indeed, By the way, I was worried by the fact they were no test on that.